### PR TITLE
First draft of the mentorship program page

### PR DESCRIPTION
**Pull request by ewels**
Feedback welcome! Need to get moving on this ASAP as the application is meant to open.. today 👀 

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1043"></a>

### CODE REVIEW

**Comment by ewels** [comment]
Added to the nav under _About_:

image

**Comment by mashehu** [comment]
I find the start a bit confusing. I was expecting to get an explanation of what the mentorship program is in the first few lines, not where the money for it comes from. 

Otherwise looks good. Should we prepare to have that also as a banner on our website when applications are opening?

**Comment by ewels** [comment]
Yes good point, it's missing an initial introduction paragraph 👍🏻 

A banner would be nice, but we can take that in a separate PR I think..

File: markdown/mentorships.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  - Testing and scaling up the use of nf-core pipelines
+- Learning how to make a contribution to nf-core
+  - Developing new and / or contributing to existing pipeline(s)
+  - Working with nf-core/tools or the website
+  - Presenting a nf-core/bytesize talk about a chosen topic
+
+### Prerequisites
+
+- Mentees
+  - Required: Some familiarity with the linux command line
```
Comment (ggabernet) [inline markdown/mentorships.md:58]:
```suggestion
  - Required: familiarity with the linux command line, git and github
```

File: markdown/mentorships.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  - Suggested: Basic bioinformatics experience
+- Mentors
+  - Required: Good experience with running Nextflow pipelines
```
Comment (ggabernet) [inline markdown/mentorships.md:61]:
```suggestion
  - Required: Good experience with running and developing Nextflow pipelines
  - Required: Experience with container engines (docker, singularity)
```

File: markdown/mentorships.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  - Suggested: Experience with developing Nextflow pipelines
```
Comment (ggabernet) [inline markdown/mentorships.md:62]:
```suggestion
```

File: markdown/mentorships.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
+## Introduction
+
+In 2022, Nextflow and nf-core are starting a joint program of mentorships.
+We hope to organise a small number of mentoring pairs to improve outreach and training in underrepresented groups and areas.
+Mentors will recieve a small stipend in return for helping their mentee.
```
Comment (jfy133) [inline markdown/mentorships.md:5]:
```suggestion
Mentors will receive a small stipend in return for helping their mentee.
```

**Review by ggabernet — COMMENTED** [review]
Hi @ewels , 
the page reads really well. I was just wondering about a bit stricter requirements for both mentors and mentees so that the program can really focus on the Nextflow and nf-core project contributions and not basic command line / git / github training. What do you think?

File: markdown/mentorships.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+Some typical aims could be:
+
+- Setting up Nextflow and nf-core in your institution
+  - Getting requirements up and running on the local compute infrastructure
+  - Developing an institutional config for you and others to use
+  - Testing and scaling up the use of nf-core pipelines
+- Learning how to make a contribution to nf-core
+  - Developing new and / or contributing to existing pipeline(s)
+  - Working with nf-core/tools or the website
+  - Presenting a nf-core/bytesize talk about a chosen topic
```
Comment (jfy133) [inline markdown/mentorships.md:53]:
```suggestion
  - Presenting a nf-core/bytesize talk about a chosen topic
  - Writing an extensive tutorial about under-described aspects of nf-core 
```

**Review by jfy133 — APPROVED** [review]
one typo, and a couple of suggests.

I still feel maybe that the intro paragrpah should be fleshed out a bit, maybe with some more concrete examples of what the pairs would be doing. 

I wonder if it's so terse people will still be a less interested to read the rest of the page.

I also wonder if the CZI Grants section is completely necessary, if this is meant to be an 'advertising page' giving this history about the grant and the 'original purpose' and how you've changed it is not that useful for applicants really, and might again put people off reading the rest of it the page... but that's just my own opinion of someone who skim read it,

**Comment by ewels** [comment]
Thanks all! I've shuffled the order of the text around a bit, hopefully it reads more good now. I think it's important to mention where the cash comes from, but I agree that we probably shouldn't lead with this (I copied it from another document to get started with writing this page, which is why it was there).

**Review by ggabernet — APPROVED** [review]
A couple minor corrections, and then looks good to me!

File: markdown/mentorships.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+### Prerequisites
+
+- Mentees
+  - Required: Familiarity with the linux command line, Git and GitHub
+  - Suggested: Basic bioinformatics experience
+- Mentors
+  - Required: Good experience with running and developing Nextflow pipelines
+  - Required: Experience with container engines (docker, singularity, ...)
+  - Suggested: Experience of contributing to the nf-core project
```
[the remark this answers is not in the document]
Comment (ewels) [reply markdown/mentorships.md:52]:
```suggestion
  - Suggested: Previous contributions to the nf-core project
```

**Pull request merged**